usage/camera/raspicam/raspiyuv.md

```diff
@@ -1,7 +1,10 @@
 # raspiyuv
 
-`raspiyuv` has the same set of features as `raspistill` but instead of outputting standard image files such as `.jpg`s, it generates raw unprocessed image files from the camera.
+`raspiyuv` has the same set of features as `raspistill` but instead of outputting standard image files such as `.jpg`s, it generates YUV420 or RGB888 image files from the output of the camera ISP.
 
-## Full documentation
+In most cases using `raspistill` is the best option for standard image capture, but using YUV can be of benefit in certain circumstances, for example, if you just need a uncompressed black and white image for computer vision applications, you can simply use the Y channel of a YUV capture.
```

Comment:
I'd probably remove the comma after "example"? In fact, might be worth splitting "For example ..." into a separate sentence?